### DISCUSSION TITLE
Turn off mujoco warning exceptions within a scope

### DISCUSSION THREAD

**Discussion by vikashplus** [google-deepmind/mujoco#651] [General]
MuJoCo_Py has has class called "[ignore_mujoco_warnings](https://github.com/openai/mujoco-py/blob/master/mujoco_py/builder.py#L376)" to turn off warnings which is quite helpful for large batched rollouts. Is there anything similar in official mujoco bindings? If not, what will be the suggested way to go about something similar? 

@yuvaltassa, @saran-t ?

**Comment by yuvaltassa** [comment]
What's the context in which you are asking this? [`mujoco.rollout`](https://github.com/deepmind/mujoco/blob/main/python/mujoco/rollout.py)?
